lib/dconf.nix

```diff
@@ -0,0 +1,15 @@
+{ lib }:
+
+# This module contains helpers for the `programs.dconf` NixOS module
+
+rec {
+  types = {
+    tuple = "_tuple";
+  };
+
+  mkTuple = first: second: {
```

Comment:
[Tuples in GVariant](https://docs.gtk.org/glib/gvariant-format-strings.html#tuples) support other lengths than pairs. Maybe looks at home-manager for inspiration: https://github.com/nix-community/home-manager/blob/886675991b643b701a33f533443db165c70692d1/modules/lib/gvariant.nix#L129